spec/rubocop/cop/style/empty_case_condition_spec.rb

```diff
@@ -266,67 +266,23 @@ def example
       let(:source) do
         <<~RUBY
           # example.rb
-          do_some_work case
-                       ^^^^ Do not use empty `case` condition, instead use an `if` expression.
-                       when object.nil?
-                         Object.new
-                       else
-                         object
-                       end
+          case
+          ^^^^ Do not use empty `case` condition, instead use an `if` expression.
+          when object.nil?
+            Object.new
+          else
+            object
+          end
         RUBY
       end
       let(:corrected_source) do
         <<~RUBY
           # example.rb
-          do_some_work if object.nil?
-                         Object.new
-                       else
-                         object
-                       end
-        RUBY
-      end
-
-      it_behaves_like 'detect/correct empty case, accept non-empty case'
-    end
-
-    context 'when using `when ... then` in `case` in `return`' do
-      let(:source) do
-        <<~RUBY
-          return case
-                 ^^^^ Do not use empty `case` condition, instead use an `if` expression.
-                 when object.nil? then Object.new
-                 else object
-                 end
-        RUBY
-      end
-      let(:corrected_source) do
-        <<~RUBY
-          return if object.nil?
-           Object.new
-                 else object
-                 end
```

Comment:
This test case is removed because this autocorrected syntax is invalid. In other words, it's a bug.